README.md

```diff
@@ -1,28 +1,35 @@
-<div align='center'>
-
-```
-██████╗  ██████╗ ██████╗     ██╗  ██╗
-██╔══██╗██╔═══██╗██╔══██╗    ██║  ██║
-██████╔╝██║   ██║██║  ██║    ███████║
-██╔═══╝ ██║   ██║██║  ██║    ╚════██║
-██║     ╚██████╔╝██████╔╝         ██║
-╚═╝      ╚═════╝ ╚═════╝          ╚═╝
-```
-
-</div>
-
-## Overview :sparkles:
-- 
-
-## Development :computer:
-![GitHub](https://img.shields.io/github/license/LaurierComputingSociety/Pod2) 
-![GitHub pull requests](https://img.shields.io/github/issues-pr/LaurierComputingSociety/Pod2)
-
-### Requirements
-- 
-
-### Setup
-```sh
-$
-```
-
+<div align='center'>
+
+```
+██████╗  ██████╗ ██████╗     ██╗  ██╗
+██╔══██╗██╔═══██╗██╔══██╗    ██║  ██║
+██████╔╝██║   ██║██║  ██║    ███████║
+██╔═══╝ ██║   ██║██║  ██║    ╚════██║
+██║     ╚██████╔╝██████╔╝         ██║
+╚═╝      ╚═════╝ ╚═════╝          ╚═╝
+```
+
+</div>
+
+## Overview :sparkles:
+- 
+
+## Development :computer:
+![GitHub](https://img.shields.io/github/license/LaurierComputingSociety/Pod2) 
+![GitHub pull requests](https://img.shields.io/github/issues-pr/LaurierComputingSociety/Pod2)
+
+### Requirements
+- 
+
+### Setup
+```api
+1.First install python, mongodb, django, djongo, in your local machine
+2.As we are using Mongodb as database so install with version command is pip install pymongo==3.12.3
+3. created superuser with
+email: optimeet123@gmail.com
+password: optimeet123
+username: admin

```

Comment:
Similar to the `settings.py` file, these secret items shouldn't be saved to GitHub due to security concerns. Please remove them and mention them in our document with the rest of the secrets. Ideally a new password should also be created since this one will be permanently saved in the history.